tools/clang/lib/Sema/SemaHLSL.cpp

```diff
@@ -10063,7 +10063,29 @@ void hlsl::HandleDeclAttributeForHLSL(Sema &S, Decl *D, const AttributeList &A,
     // The attribute has been set but will have no effect. Validation will emit a diagnostic
     // and prevent code generation.
     ValidateAttributeTargetIsFunction(S, D, A);
+
+    return; // SPIRV Change
+  }
+
+  // SPIRV Change Starts
```

Comment:
I wonder whether this code and some of the other code above should also be wrapped in `#ifdef ENABLE_SPIRV_CODEGEN` ?